src/cpu/kvm/base.cc

```diff
@@ -690,7 +706,9 @@ BaseKvmCPU::kvmRunDrain()
 uint64_t
 BaseKvmCPU::getHostCycles() const
 {
-    return hwCycles.read();
+    if (usePerf)
+        return hwCycles->read();
+    return 0;
```

Comment:
Should we have a warning here if not using perf? Or, would that warning trigger too often?

Reply:
Done here

https://github.com/gem5/gem5/pull/95/commits/9c3dc593a982c302b310b4dc648f6be2778fcad3#diff-59e4d8f93176a7efcdc59ddf19d7d54d4fac3571603dd4dd389281b18db06860R112